UI updates

Joseph-Cashmore opened the pull request:
Styling changes due to font-size change.

Other small updates were made too.

Commit:
font size adjustments

Commit:
padding adjustments

Commit:
updates to "completed" styling

I couldn't find a scenario with the use of a line separator that I liked, the heading is better.

Commit:
made new item form text more generic

Commit:
test fixes

Joseph-Cashmore committed:
Merge branch 'main' into ui-updates

samstradling commented on src/components/ListItems/ListItems.js:
```diff
@@ -17,7 +17,7 @@ const ListItems = ({
       />
     ))}
-    <h3>Added items</h3>
+    <h4 className={styles.completedHeading}>Completed</h4>
```
I'm a little worried about this text colour standing out against the white:
https://coolors.co/contrast-checker/fabe0e-ffffff

Joseph-Cashmore replied:
Ooh that's an interesting link, will come in handy.

Yes I think you're right, I tried a few different variations and haven't found a good fit yet. This felt like a "for now" decision. I will explore some more ideas and see what I can come up with!

samstradling commented on src/components/ListItem/styles.module.css:
```diff
@@ -12,12 +11,16 @@
+  font-weight: 600;
   flex-grow: 1;
   cursor: pointer;
+  color: var(--font_grey);
```
I think this grey on grey text makes it quite faint

Joseph-Cashmore replied:
I will make the font grey color darker so that there's a clearer distinction.

Commit:
darker font for items

For some reason my laptop would no longer get this running without converting every file from CLF to LF...

Commit:
padding for different device viewports

Not sure what effect this will have...added in some extra padding on the list selector for good measure

Commit:
Completed heading style change

I've had another go at the completed heading -- I've tried to use the same style as the logo, and this can be reflected in the save for later heading too

samstradling commented on public/index.html:
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="viewport-fit=cover" />
```
I think this has made the page massive (and therefore everything tiny) on my phone, has it done the same for you?

Commit:
remove viewport fit cover

Commit:
A few margin changes

Commit:
I can't add (added margin to base of wrapper)

Ensures the lats item can always be seen
